src/Cell.sublime-workspace

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
Please exclude this file. You can add `src/Cell.sublime-workspace` to `.gitignore`.